src/JSTest/RuntimeProviders/NodeRuntimeProvider.cs

```diff
@@ -36,6 +36,12 @@ public TestProcessStartInfo GetRuntimeProcessInfo(string nodePath, bool isDebugE
                 processInfo.FileName = "node.exe";
             }
 
+            if (string.IsNullOrWhiteSpace(rootFolder))
+            {
+                rootFolder = Path.GetDirectoryName(typeof(TestRunner).GetTypeInfo().Assembly.GetAssemblyLocation());
```

Comment:
where is this finally used? also shouldn't rootFolder by default point to the test repo and not jstestadapter?